src/webextension/widgets/scrolling-list.css

```diff
@@ -16,10 +16,14 @@
 
 .scrolling-list > li[data-selected] {
   color: #0c0c0d;
-  background-color: #b1b1b3;
+  background-color: #f3f3f5;
+  border-right: none;
 }
 
 .scrolling-list:focus > li[data-selected] {
-  color: #ffffff;
-  background-color: #0060df;
+  box-shadow: inset 0 0 0 2px #0a84ff, 0 0 0 3px rgba(10, 132, 255, 0.3);
```

Comment:
This looks really broken on my end. The outset shadow doesn't show up properly, and the inset shadow gets cut off by the bottom border on the elements. I'm really not sure how we can fix this in a way that follows our visual spec while retaining some sort of focus state for keyboard users.